environment_and_metadata.rst

```diff
@@ -184,18 +183,18 @@ environment variables for correct operation of most software.
 
 .. code::
 
-   $ singularity exec --cleanenv env.sif env
+   $ apptainer exec --cleanenv env.sif env
    HOME=/home/dave
    LANG=C
-   LD_LIBRARY_PATH=/.singularity.d/libs
+   LD_LIBRARY_PATH=/.apptainer.d/libs
    PATH=/startpath:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
    PROMPT_COMMAND=PS1="Singularity> "; unset PROMPT_COMMAND
    PS1=Singularity>
    PWD=/home/dave/doc-tesrts
-   SINGULARITY_COMMAND=exec
-   SINGULARITY_CONTAINER=/home/dave/doc-tesrts/env.sif
-   SINGULARITY_ENVIRONMENT=/.singularity.d/env/91-environment.sh
-   SINGULARITY_NAME=env.sif
+   apptainer_COMMAND=exec
+   apptainer_CONTAINER=/home/dave/doc-tesrts/env.sif
+   apptainer_ENVIRONMENT=/.apptainer.d/env/91-environment.sh
+   apptainer_NAME=env.sif
```

Comment:
Keep these environment variables as `SINGULARITY_` for now, same comment about `/.apptainer.d` -> `/.singularity.d`